frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableDetail/TableDetail.module.css

```diff
@@ -1,11 +1,14 @@
 .wrapper {
   display: grid;
   grid-template-rows: auto 1fr;
-  width: 500px;
+  width: 300px;
   height: 100%;
   border-left: 1px solid var(--pane-border, #383a3b);
   background-color: var(--pane-background, #232526);
   box-shadow: 0px 4px 20px 0px var(--shadow-basic-shadow, #000);
+  word-break: break-all;
+  list-style-position: outside;
+  margin-left: 12px;
```

Comment:
Q: Is this necessary?

Reply:
`margin-left` effects are indeed in place, but it doesn't seem to be having any visual impact at the moment. I'll remove it!